airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -16,33 +16,34 @@
     ],
     "additionalProperties": false,
     "properties": {
-      "s3_endpoint": {
-        "title": "Endpoint",
+      "access_key_id": {
```

Comment:
For all of these fields, where are the relevant docs linked so a user can find more information as part of this checklist item?

Reply:
Added links for required and optional fields.